docs/tutorial/force_calibration.rst

```diff
@@ -1,380 +1,440 @@
 
-.. math::
+    print(len(force_slice.calibration))
 
-    R_d = \sqrt{\frac{D_\mathrm{physical}}{D_\mathrm{measured}}} \quad \mathrm{[m/V]}
+Next we'll decalibrate the force data back to the original voltage measured by the PSD. Let's write a short function for this so we can reuse it elsewhere::
 
-Both of these quantities depend on the parameter :math:`\gamma_0`, which corresponds to the drag coefficient of a sphere and is given by:
+    def decalibrate(force_slice):
+        offset = force_slice.calibration[0]["Offset (pN)"]
+        response = force_slice.calibration[0]["Rf (pN/V)"]
+        return (force_slice - offset) / response
 
-.. math::
+    volts = decalibrate(force_slice)
 
-    \gamma_0 = 3 \pi \eta d \quad \mathrm{[kg/s]}
+Performing the calibration
+--------------------------
 
-where :math:`\eta` corresponds to the dynamic viscosity [Pa*s] and :math:`d` is the bead diameter [m].
-Note that in `pylake` we actually use `microns` to specify the bead diameter for convenience.
+For convenience, Pylake offers :func:`~lumicks.pylake.calibrate_force()` which performs an entire calibration procedure in a single function call::
 
-We use the bead diameter found in the calibration performed in Bluelake.
-You can optionally also provide a viscosity (in Pa/s) and temperature (in degrees Celsius)::
+    calibration = lk.calibrate_force(
+        volts.data,
+        force_slice.calibration[0]["Bead diameter (um)"],  # Use the value from the last calibration
+        temperature=25,
+        sample_rate=volts.sample_rate
+    )
 
-    bead_diameter = f.force1x.calibration[1]["Bead diameter (um)"]
-    force_model = lk.PassiveCalibrationModel(bead_diameter, viscosity=0.001002, temperature=20)
+This function allows us to quickly try various calibration settings without having to set up all the intermediate models and spectra explicitly.
+Note that most parameters have to be provided as keyworded arguments to prevent errors.
+This method returns a :class:`~lumicks.pylake.force_calibration.power_spectrum_calibration.CalibrationResults`, which can be plotted using :func:`~lumicks.pylake.force_calibration.power_spectrum_calibration.CalibrationResults.plot()`.
 
-To find the viscosity of water at a particular temperature, you can use :func:`~lumicks.pylake.viscosity_of_water` :cite:`huber2009new`.
-When omitted, this function will automatically be used to look up the viscosity of water for that particular temperature.
-To fit this model to the data, you can now invoke::
+The rest of this tutorial illustrates the various steps involved when performing such a calibration.
 
-    calibration = lk.fit_power_spectrum(power_spectrum, force_model)
-    calibration
+Obtaining the power spectrum
+----------------------------
 
-This will produce a table with your fitted calibration parameters.
+Next, we need the power spectrum to fit. To compute a power spectrum from our data we can invoke :func:`~lumicks.pylake.calculate_power_spectrum()`::
```

Comment:
considering the change above, "next" doesn't make sense in this context. We could do something like:
```suggestion
To use the more manual lower-level API, we first need the power spectrum to fit. To compute a power spectrum from our data we can invoke :func:`~lumicks.pylake.calculate_power_spectrum()`::
```